include/revng/StackAnalysis/AAWriterPass.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/IR/AssemblyAnnotationWriter.h"
+#include "llvm/IR/PassManager.h"
+
+#include "revng/Support/revng.h"
+
+class AAWriterPass : public llvm::PassInfoMixin<AAWriterPass> {
+  const bool StoresOnly;
+
+public:
+  AAWriterPass(bool StoresOnly = false) : StoresOnly(StoresOnly) {}
```

Comment:
Are you actually using this argument somewhere? If not, drop.

Reply:
I'm passing it to its implementation, it is necessary, I can't drop it. I found analyzing only the stores useful, back when fixing AA issues.